src/DependencyInjection/HWIOAuthExtension.php

```diff
@@ -46,6 +46,8 @@ function service($class): ReferenceConfigurator
  */
 final class HWIOAuthExtension extends Extension
 {
+    private array $firewallNames = [];
```

Comment:
```suggestion
    /**
     * @var array<string, true>
     */
    private array $firewallNames = [];
```

Reply:
Additionally I must say I really don't like this here, IMHO it's wrong to inject whole bundle extension, I would prefer to create some internal new class like:
```php
namespace Symfony\Component\DependencyInjection\Loader\Configurator;

/**
 * @internal
 */
final class SecurityFirewallStorage extends \ArrayIterator
{}